projects/Mallard/src/screens/settings/dev-zone.tsx

```diff
@@ -57,9 +81,9 @@ const DevZone = withNavigation(({ navigation }: NavigationInjectedProps) => {
     }, [])
 
     useEffect(() => {
-        RNFetchBlob.fs
-            .ls(FSPaths.issuesDir + '/daily-edition')
-            .then(files => setFiles(JSON.stringify(files)))
+        getFileList().then(fileList => {
+            setFiles(beautify_js(JSON.stringify(fileList)))
```

Comment:
Interesting fact `JSON.format(fileList, null, 2)` might do the same job as this but not worth worrying about now 🆗